be1-go/cli/pop.go

```diff
@@ -10,6 +11,41 @@ import (
 )
 
 func main() {
+	publicKeyFlag := &cli.StringFlag{
+		Name:    "public-key",
+		Aliases: []string{"pk"},
+		Usage:   "base64url encoded server's public key",
+	}
+	organizerAddressFlag := &cli.StringFlag{
+		Name:    "organizer-address",
+		Aliases: []string{"org"},
+		Usage:   "ip address of organizer",
+		Value:   "localhost",
+	}
+	organizerPortFlag := &cli.IntFlag{
+		Name:    "organizer-port",
+		Aliases: []string{"op"},
+		Usage:   "port on which to connect to organizer websocket",
+		Value:   9002,
+	}
+	clientPortFlag := &cli.IntFlag{
+		Name:    "client-port",
+		Aliases: []string{"cp"},
+		Usage:   "port to listen websocket connections from clients on",
+		Value:   9000,
+	}
+	witnessPortFlag := &cli.IntFlag{
+		Name:    "witness-port",
+		Aliases: []string{"wp"},
+		Usage:   "port to listen websocket connections from witnesses on",
+		Value:   9002,
```

Comment:
This should be a different value than `9002` because it conflicts with the default for `organizerPortFlag`